.github/workflows/RunTests.yml

```diff
@@ -60,14 +60,26 @@ jobs:
       device_name: a100-40gb-4
       build_mode: stable_stack
       base_image: us-docker.pkg.dev/tpu-prod-env-multipod/jax-stable-stack/candidate/gpu:latest
+  
+  cpu_unit_tests:
+    needs: tpu_image
+    uses: ./.github/workflows/run_tests_internal.yml
+    with:
+      device_type: tpu
```

Comment:
I don't think this runs on a CPU machine currently. If you see the test runs, the machine chosen is "Runner name: 'shralex-github-runner-1'" which is a TPU machine